File: app/src/main/java/com/example/helloworld/CountryScreen.java
package com.example.helloworld;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.appcompat.app.AppCompatActivity;

public class CountryScreen extends AppCompatActivity {

    private Button buttonPoland;
    private Button buttonChina;
    private Button buttonItaly;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_country_screen);

        buttonPoland = (Button) findViewById(R.id.poland);
        buttonChina = (Button) findViewById(R.id.china);
        buttonItaly = (Button) findViewById(R.id.italy);
        buttonPoland.setOnClickListener(new View.OnClickListener(){
            @Override
            public void onClick(View view){
                openPolandActivity();
            }
        });
        buttonChina.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                openChinaActivity();
            }
        });
        buttonItaly.setOnClickListener(new View.OnClickListener(){
            @Override
            public void onClick(View view){
                openItalyActivity();
            }

        });
    }

    public void openPolandActivity() {
        Intent intent = new Intent(this, PolandChartScreen.class);
        startActivity(intent);
    }
    public void openChinaActivity() {
        Intent intent = new Intent(this, ChinaChartScreen.class);
        startActivity(intent);
    }
    public void openItalyActivity(){
        Intent intent = new Intent(this, ItalyChartScreen.class);
        startActivity(intent);
    }


}

File: app/src/main/java/com/example/helloworld/CountryScreen.kt
package com.example.helloworld

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class CountryScreen : AppCompatActivity() {
    private var buttonPoland: Button? = null
    private var buttonChina: Button? = null
    private var buttonItaly: Button? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_country_screen)
        buttonPoland = findViewById<View>(R.id.poland) as Button
        buttonChina = findViewById<View>(R.id.china) as Button
        buttonItaly = findViewById<View>(R.id.italy) as Button
        buttonPoland!!.setOnClickListener { openPolandActivity() }
        buttonChina!!.setOnClickListener { openChinaActivity() }
        buttonItaly!!.setOnClickListener { openItalyActivity() }
    }

    fun openPolandActivity() {
        val intent = Intent(this, PolandChartScreen::class.java)
        startActivity(intent)
    }

    fun openChinaActivity() {
        val intent = Intent(this, ChinaChartScreen::class.java)
        startActivity(intent)
    }

    fun openItalyActivity() {
        val intent = Intent(this, ItalyChartScreen::class.java)
        startActivity(intent)
    }
}